htdocs/core/lib/company.lib.php

```diff
@@ -710,51 +710,25 @@ function getFormeJuridiqueLabel($code)
  */
 function getCountriesInEEC()
 {
-	global $conf;
-
-	// List of all country codes that are in europe for european vat rules
-	// List found on http://ec.europa.eu/taxation_customs/common/faq/faq_1179_en.htm#9
-	$country_code_in_EEC = array(
```

Comment:
We must also guarantee the migration. Here we have list of country in EEC but this list is not yet defined into database, so you must also:
- Edit file tdocs/install/mysql/data/llx_00_c_country.sql to add the flag to 1 when country is in EEC (for first installation)
- Add a command into the migration script into v13 to v14 to update all country code in EEC to set the value to 1 of the EEC flag (currently it is empty for all countries).